integration_tests/test_env_populator.py

```diff
@@ -0,0 +1,41 @@
+from integration_tests.conftest import test_session
+from robyn.env_populator import load_vars, parser
+import pathlib
+import os
+import pytest
+import requests
+
+
+
+path = pathlib.Path(__file__).parent  
+
+#create robyn.env before test and delete it after test
+@pytest.fixture
+def env_file():
+    CONTENT = "PORT=8080"
+    dir = path / "test_dir"
+    env_file = dir / "robyn.env"
+    env_file.write_text(CONTENT)
+    yield
+    env_file.unlink()
```

Comment:
Just realized that unlinking the file will not be enough here. We will also have to unset the environment variables.

Reply:
will do